cognite/extractorutils/uploader_extractor.py

```diff
@@ -129,6 +132,11 @@ def handle_output(self, output: CdfTypes) -> None:
         else:
             raise ValueError(f"Unexpected type: {type(peek)}")
 
+    def _apply_middleware(self, peek):
+        for mw in self.middleware:
+            peek = mw(peek)
+        return peek
```

Comment:
Minor thing, but I think `peek` here refers to looking at the contents of the iterator and deciding based on that, it's a little confusing to use it as a variable name in this method.